sdk/Sdk.Generators/FunctionMetadataProviderGenerator/FunctionMetadataProviderGenerator.Parser.cs

```diff
@@ -623,9 +623,9 @@ private bool TryGetAttributeProperties(AttributeData attributeData, Location? at
 
                 foreach (var namedArgument in attributeData.NamedArguments)
                 {
-                    if (namedArgument.Value.Value != null)
+                    if (WriteArgumentToMetadata(namedArgument.Value))
```

Comment:
What part was exactly crashing and why? The exception thrown from roslyn suggests "Use Values property." but I don't see we use the `Values` property in this PR.

Reply:
The use of `Values` is [elsewhere](https://github.com/Azure/azure-functions-dotnet-worker/blob/d45568b5ce9a84e4df509ddc132cf2a6b05e97dc/sdk/Sdk.Generators/FunctionMetadataProviderGenerator/FunctionMetadataProviderGenerator.Parser.cs#L735) outside of the PR - the problem was that we were null-checking `namedArgument.Value.Value` in this spot, which is invalid in the case that `namedArgument.Value` is an array type and caused a crash.